packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/ModelOperationCopy.java

```diff
@@ -0,0 +1,16 @@
+package software.uncharted.terarium.hmiserver.models.dataservice;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import java.io.Serializable;
+
+@Data
+@Accessors(chain = true)
+public class ModelOperationCopy implements Serializable {
+	private Long left;
+	private String name;
+	private String description;
+	private String framework;
```

Comment:
Shouldn't the `framework` be the one from the copied model?
If we want to change the framework from PetriNet to ODE for example, we should call another endpoint and not mix them.

Reply:
This is part of the minimal copy-operation payload as specified by TDS. Given we had almost no write-operations until now there are bound to be some quirky things we have missed or have become irrelevant. 

CC @mattprintz @fivegrant if there are any comments.